pnpm-lock.yaml

```diff
@@ -1,809 +1,3703 @@
-lockfileVersion: '6.0'
+lockfileVersion: '9.0'
```

Comment:
From previous experience, I am a bit concerned that after updating pnpm to v9, this change will cause some issues in the CI jobs, but I couldn't figure out how to prevent it, so it might be we'll just need to react only afterwards.

Reply:
This commit hopefully prevents potential issues: [5c0145b](https://github.com/warp-ds/css/pull/198/commits/5c0145bbeaa62aada5ed7d19d3a1a996cc2a51b2)